src/javascript/app/base/footer.js

```diff
@@ -1,4 +1,29 @@
+const BinarySocket = require('./socket');
+const Client       = require('../base/client');
+const State        = require('../../_common/storage').State;
+
 const Footer = (() => {
+    const onLoad = () => {
+        BinarySocket.wait('website_status', 'authorize').then(() => {
+            const $footer      = $('#footer-last');
+            const $cfd_warning = $footer.find('.eu-only');
+            const show_warning = (Client.isLoggedIn() ?
+                (/maltainvest/.test(State.getResponse('authorize.landing_company_name'))
+                    || /financial/.test(Client.getBasicUpgradeInfo().can_upgrade_to)
+                    || Client.hasAccountType('financial'))
```

Comment:
can we check has account type before checking can upgrade to financial? it takes less processing so it's better to check that first

Reply:
yes, sure 👍